src/libzerocoin/AccumulatorProofOfKnowledge.cpp

```diff
@@ -52,24 +52,24 @@ AccumulatorProofOfKnowledge::AccumulatorProofOfKnowledge(const AccumulatorAndPro
 	CBigNum r_sigma = CBigNum::randBignum(params->accumulatorPoKCommitmentGroup.modulus);
 	CBigNum r_xi = CBigNum::randBignum(params->accumulatorPoKCommitmentGroup.modulus);
 
-	CBigNum r_epsilon =  CBigNum::randBignum((params->accumulatorModulus/4) * CBigNum(2).pow(params->k_prime + params->k_dprime));
+    CBigNum r_epsilon =  CBigNum::randBignum(aM_4 * aR);
```

Comment:
Since it's used several times (5 actually), might be good to go further and have the result of `aM_4 * aR` stored too.

Reply:
Yeah good catch, will push it tonight.